### DISCUSSION TITLE
Backend operation: missing id after creating a user

### DISCUSSION THREAD

**Discussion by lgmohale** [apioo/fusio#551] [Q&A] [answered]
The ID isn't returned by the [create user endpoint](https://www.fusio-project.org/api/backend#tag/user/operation/backend.user.create) as stated in the documentation. Could you please help with this?

**Comment by chriskapp** [comment, marked as answer]
Hi @lgmohale this is indeed currently missing, I have just created an issue https://github.com/apioo/fusio/issues/552 and I will try to include the fix for the upcoming 5.0 release.
